client/my-sites/upgrades/domain-management/domain-connect/domain-connect-authorize-footer.jsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { translate } from 'i18n-calypso';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import { actionType } from './constants';
+
+class DomainConnectAuthorizeFooter extends Component {
+
+	placeholder = () => {
+		return (
+			<div className="domain-connect__footer is-placeholder">
+				<span></span>
+				<span></span>
+			</div>
+		);
+	}
+
+	renderActionConfirmCancel = () => {
+		return (
+			<div>
+				<p>
+					{
+						translate( 'When you\'re ready to proceed, click Confirm. If this isn\'t what you meant to do, ' +
+							'click Cancel and we won\'t make any changes.' )
+					}
+				</p>
+				<div>
+					<Button
```

Comment:
Thanks. I was actually worried that I might get dinged for doing this, but I hate spinners and "submitting" messages that flash on the screen quickly.